values/harbor-app-operator/harbor-app-operator.gotmpl

```diff
@@ -0,0 +1,29 @@
+{{- $v := .Values }}
+{{- $o := $v.apps | get "harbor-app-operator" }}
+{{- $version := $v.versions.tasks }}
+{{- $isSemver := regexMatch "^[0-9.]+" $version }}
+
+image:
+  tag: {{ printf "%s%s" ($isSemver | ternary "v" "") $version }}
+  pullPolicy: {{ $isSemver | ternary "IfNotPresent" "Always" }}
+
+{{- with $v.otomi | get "globalPullSecret" nil }}
+imagePullSecrets:
+  - name: otomi-pullsecret-global
+{{- end }}
+configs:
```

Comment:
I dont think this `config.tls.certificates` is used

Reply:
Removed!